src/components/ToDoComponent/SettingGoals/SettingGoalsComponents/Calendar.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import { Text, TouchableOpacity, StyleSheet, ScrollView } from 'react-native';
+import DatePicker from 'react-native-date-picker';
+
+const Calendar = () => {
```

Comment:
@JAYATEJA04 , do you think Calendar name is appropriate for this component, after considering what it does?

Reply:
This is done.